src/actions/guides/testing/ci-cd.cr

```diff
@@ -0,0 +1,60 @@
+class Guides::Testing::Introduction < GuideAction
```

Comment:
This class name should be `Guides::Testing::ContinuousIntegration` or something like that. We have an `Introduction` class already.